docs-library/10x-funding-stages.md

```diff
@@ -0,0 +1,75 @@
+# 10x Funding Stages
+
+*Throughout this document we will provide a description of each funding stage and what’s usually expected to move on for further funding of your project.*
+
+**10x exists to invest in products that possess paradigm-changing ideas or technology with the potential to deliver massive value (at least “10 times” the value of the initial investment) to the federal government or the American public.**
```

Comment:
This seems like different boilerplate that what we've been using on the website, should it be updated? That being said, we probably don't need to hold up this PR for that.